### PR TITLE
Add fastify-cli.config.js to custom ignored directory in watch mode

### PR DESCRIPTION
**Pull request by gaogao1030**
Ignore configurable for watch mode

### CODE REVIEW

**Comment by cemremengu** [comment]
Related to #26

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ You can pass the following options via cli arguments, every options has the corr
 | Socket to listen on | `-s` | `--socket` | `FASTIFY_SOCKET` |
 | Log level (default to fatal) | `-l` | `--log-level` | `FASTIFY_LOG_LEVEL` |
 | Prints pretty logs | `-P` | `--pretty-logs` | `FASTIFY_PRETTY_LOGS` |
-| Watch process.cwd() directory for changes, recursively; when that happens, the process will auto reload. | `-w` | `--watch` | `FASTIFY_WATCH` |
+| Watch process.cwd() directory for changes, recursively; when that happens, the process will auto reload. more detail for [fastify-cli.config.js](#fastify-cliconfigjs)| `-w` | `--watch` | `FASTIFY_WATCH` |
```
Comment (cemremengu) [inline README.md]: This should read `See [fastify-cli.config.js](#fastify-cliconfigjs) for more details`

File: README.md [hunk 2]
```diff
@@ -198,6 +198,16 @@ It is a standard Fastify plugin and you will not need to add the `listen` method
 },
 ```
 
+### fastify-cli.config.js
+If you want to use custom config file, just set process.env.FASTIFY_CLI_CONFIG = path/to/file.
+(The default is fastify-cli.config.js in the process.cwd() directory.)
+
+```js
+module.exports = {
+  ignore: /(node_modules|\.git|bower_components|build|dist|pages|\.next)/
```
Comment (cemremengu) [inline README.md]: Maybe we can add `logs` to this list as well ?

**Comment by mcollina** [comment]
I think this should not be using a config file. We are using dotenv and env variables for in this project and I would prefer to not have two different config mechanism.

**Comment by cemremengu** [comment]
@gaogao1030 Maybe we can just add `--ignore-watch="logs run"` (smilar to pm2) option and the corresponding ENV variables? What do you think?

**Comment by gaogao1030** [comment]
> @gaogao1030 Maybe we can just add `--ignore-watch="logs run"` (smilar to pm2) option and the corresponding ENV variables? What do you think?

Can we specify a configuration file with an environment variable?  like `pm2 start ecosystem.config.js
`

**Comment by cemremengu** [comment]
@gaogao1030 I think so yes. Checkout http://pm2.keymetrics.io/docs/usage/environment/

**Pull request closed without merging**